public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -290,23 +394,27 @@ export class SurveyAnswerReactComponent extends React.Component<Props> {
                     </div>
                     <div className='survey-tbl-r-group survey-tbl-r-group-style '>
                         <div className='info-text survey-tbl-d display-body'>
-                            {isCopeSurvey ? <h1>testIs cope</h1> : <span><React.Fragment>Concept Code</React.Fragment>
+                        {isCopeSurvey ? null :
+                            <span><React.Fragment>Concept Code</React.Fragment>
                                 <TooltipReactComponent tooltipKey='conceptCodeHelpText' label='test' searchTerm='test' action='Survey Page Tooltip' /></span>
```

Comment:
No need for `<React.Fragment>` around the text here.

Nit: move text to new line, fix indentation:
```suggestion
                        {isCopeSurvey ? null :
                            <span>
                                Concept Code
                                <TooltipReactComponent tooltipKey='conceptCodeHelpText' label='test' searchTerm='test' action='Survey Page Tooltip' />
                            </span>
```